### PR TITLE
fix message popup

### PR DESCRIPTION
**Pull request by didOstap**

### CODE REVIEW

File: client/src/components/Alert/Alert.tsx [hunk 1]
```diff
@@ -18,19 +18,34 @@ interface IAlertState {
...
-    <UncontrolledAlert color={color} onclick={(): void => {cleanAlert(); }}>{message}</UncontrolledAlert>
+                                                 message,
+                                                 color,
+                                                 // tslint:disable-next-line:no-shadowed-variable
+                                                 clearAlert,
+                                             }: IAlertProps): any => message &&
+    <UncontrolledAlert color={color} onClick={(): void => {
+        clearAlert();
+    }}>
+        {message} {timer(clearAlert)} {clear(timer)}</UncontrolledAlert>;
```
Comment (InnaKutsik) [inline client/src/components/Alert/Alert.tsx]: {message} {timer(clearAlert)} ?????

File: client/src/components/Alert/Alert.tsx [hunk 2]
```diff
@@ -1,36 +1,28 @@
...
+import { FormProps, UncontrolledAlert } from 'reactstrap';
 import { clearAlert } from '../../store/alert/actions';
-import { UncontrolledAlert } from 'reactstrap';
 
-interface IAlertProps {
-    message: string;
-    color: string;
-    cleanAlert: any;
-}
+export default class Alert extends React.Component<any> {
```
Comment (InnaKutsik) [inline client/src/components/Alert/Alert.tsx]: any?

File: client/src/components/Alert/AlertContainer.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import * as actions from '../../store/alert/actions';
+import Alert from './Alert';
+import { connect } from 'react-redux';
+
+const AlertContainer: React.FunctionComponent<any> = ({
+                                                          message,
+                                                          color,
+                                                          clearAlert,
+                                                      }: any): JSX.Element | null => !!message ? (
```
Comment (InnaKutsik) [inline client/src/components/Alert/AlertContainer.tsx]: any?

File: client/src/components/Alert/AlertContainer.tsx [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+    <Alert clearAlert={clearAlert} message={message} color={color}/>
+) : null;
+
+const mapStateToProps = (state: any): { message: string, color: string } => ({
```
Comment (InnaKutsik) [inline client/src/components/Alert/AlertContainer.tsx]: any?

File: client/src/components/Alert/Alert.tsx (hunk 2 above)
Comment (didOstap) [reply client/src/components/Alert/Alert.tsx]: fixed

File: client/src/components/Alert/AlertContainer.tsx (hunk 3 above)
Comment (didOstap) [reply client/src/components/Alert/AlertContainer.tsx]: fixed

**Pull request merged**